tests/container-copy

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+set -eu
+
+# Install and setup LXD.
+install_lxd
+lxd init --auto
+
+cp /root/go/bin/lxd /var/snap/lxd/common/lxd.debug
+systemctl reload snap.lxd.daemon
+lxd waitready --timeout 100
+
+# Create a loop device that we can mount as a source for a dir storage pool.
+test_dir="$(mktemp -d XXX.test)"
+loopimg="$(mktemp "${test_dir}/XXX.img")"
+truncate -s 5G "${loopimg}"
+loopdev="$(losetup --show --find "${loopimg}")"
+mkfs.xfs "${loopdev}"
```

Comment:
I've added a loop that checks `xfs` `ext4` and `btrfs`. 

For `ext4` we also have to delete `lost+found` otherwise LXD will complain that the source is non-empty.